src/helpers/cellStyle.ts

```diff
@@ -0,0 +1,4 @@
+export function cellStyle(width: number | undefined) {
```

Comment:
between this and `ColumnHeader.helpers.ts` being very short, I think I would prefer a single `helpers.ts` file.